v-next/hardhat-utils/test/helpers/synchronization.ts

```diff
@@ -0,0 +1,8 @@
+import { MultiProcessMutex } from "../../src/synchronization.js";
+
+const m = new MultiProcessMutex("test-mutex", 20_000);
```

Comment:
This will break if the mutex name in `v-next/hardhat-utils/test/synchronization.ts` changes, and it’s easy to miss. Could we define a const in that file and import it here?